### PR TITLE
Update RCTP

### PR DESCRIPTION
**Pull request by JA802T**
http://eaip.caa.gov.tw/eaip/history/2018-11-08-AIRAC/html/index-en-TW.html

### CODE REVIEW

File: Fixes.json [hunk 1]
```diff
@@ -405944,6 +405990,11 @@
...
     "Longitude": 2.750278
   },
+  {
+    "Latitude": 27.091666667,
+    "Name": "BULAN",
+    "Longitude": 124
+  },
   {
     "Latitude": 47.0997238,
     "Name": "BULAN",
```
[the remark this answers is not in the document]
Comment (ItsW1ll) [reply Fixes.json]: Here you just put 124. It should be at least 124.0

[the remark this answers is not in the document]
Comment (JA802T) [reply Fixes.json]: okay

File: Fixes.json [hunk 2]
```diff
@@ -456929,6 +456985,11 @@
...
     "Longitude": -122.048744
   },
+  {
+    "Latitude": 25.205,
+    "Name": "DECOY",
+    "Longitude": 121.01833333
+  },
   {
     "Latitude": 41.7396736,
     "Name": "DECOY",
```
[the remark this answers is not in the document]
Comment (JA802T) [reply Fixes.json]: How to fix it?

File: Fixes.json [hunk 3]
```diff
@@ -405944,6 +405990,11 @@
     "Name": "BULAM",
```
[the remark this answers is not in the document]
Comment (JA802T) [reply Fixes.json]: i didn't touch BULAM...

**Pull request merged**